libraries/AP_Gripper/AP_Gripper.cpp

```diff
@@ -58,12 +58,12 @@ const AP_Param::GroupInfo AP_Gripper::var_info[] = {
     // @Units: s
     AP_GROUPINFO("REGRAB",  5, AP_Gripper, config.regrab_interval, GRIPPER_REGRAB_DEFAULT),
 
-    // @Param: UAVCAN_ID
+    // @Param: CAN_ID
     // @DisplayName: EPM UAVCAN Hardpoint ID
     // @Description: Refer to https://docs.zubax.com/opengrab_epm_v3#UAVCAN_interface
```

Comment:
spotted a couple of missed UAVCAN refences

Reply:
intentionally did not change those.....the link is external, cant be changed.....and when someone follows it, wanted at least the same nomenclature to match the reference...same with some Mission Planner references...